packages/synapse-react-client/src/components/DataGrid/columns/AutocompleteColumn.test.tsx

```diff
@@ -51,8 +53,16 @@ describe('AutocompleteColumn', () => {
     // Check that the input displays the current value
     expect(input).toHaveValue('freeText')
 
-    await userEvent.tab() // Simulate blur by tabbing away
-
+    // Simulate blurring by changing active/focus to false
```

Comment:
The focus state is technically 'controlled' by react-datasheet-grid, so to unfocus, we have to change the props.